aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/RealAWSCognitoAuthPlugin.kt

```diff
@@ -127,7 +127,9 @@ internal class RealAWSCognitoAuthPlugin(
         onError: Consumer<AuthException>
     ) {
         authStateMachine.getCurrentState { authState ->
-            if (authState.authNState is AuthenticationState.Configured) {
+            if (authState.authNState is AuthenticationState.Configured ||
```

Comment:
I still don't think this is quite right. Ex: if were are not in a Configured or SignedOut state, we will throw an error that Amplify configuration file is problematic which really isn't the case.

Reply:
yeah we should throw InvalidStateException.